backend/controllers/Task/IndexController.js

```diff
@@ -0,0 +1,14 @@
+class IndexController {
+    constructor(taskService, httpStatusCodes) {
+        this.taskService = taskService;
+        this.httpStatusCodes = httpStatusCodes;
+    }
+
+    async invoke(req, res) {
+        const tasks = await this.taskService.findAll({ user_id: request.session.user.id });
+
+        return res.status(this.httpStatusCodes.OK).json({ tasks });
```

Comment:
res.send({ task })
also please fix responses and do not nest them, it should be `send(tasks)`

Reply:
done